Gemfile

```diff
@@ -41,6 +41,10 @@ group :development, :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '~> 2.13'
   gem 'selenium-webdriver'
+  gem 'rspec-rails', '~> 3.7.2'
```

Comment:
Make sure you install rubocop and ESLint: 
https://connect.we.co/display/DIGI/Linter+Setup

It will make sure your Ruby is written based on convention. For example, gems should be written in alphabetical order